base/Dockerfile

```diff
@@ -8,11 +8,13 @@ MAINTAINER Sébastien Han "seb@redhat.com"
 ENV CEPH_VERSION firefly
 
 # Install prerequisites
-RUN apt-get update
-RUN apt-get install -y wget
+RUN apt-get update && apt-get install -y wget bash
 
 # Install Ceph
-CMD wget -q -O- 'https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/release.asc' | apt-key add -
+RUN wget -q -O- 'https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/release.asc' | apt-key add -
 RUN echo deb http://ceph.com/debian-$CEPH_VERSION/ trusty main | tee /etc/apt/sources.list.d/ceph-$CEPH_VERSION.list
-RUN apt-get update
-RUN apt-get install -y --force-yes ceph
+RUN apt-get update && apt-get install -y --force-yes ceph
+
+COPY ./docker-entrypoint.sh /docker-entrypoint.sh
```

Comment:
I would prefer to keep the entrypoint script names the same, for consistency.  Otherwise, the downstream images will end up having multiple entrypoint scripts, which becomes confusing.

TLDR; rename to `/entrypoint.sh`